input/pagecontent/conformance.md

```diff
@@ -1,10 +1,135 @@
+
+    ```
+    {
+      "resourceType": "CapabilityStatement",
+      ...
+      "instantiates": [
+        "http://hl7.org.au/fhir/ereq/CapabilityStatement/au-erequesting-server"
+      ],
+      ...  
+      "rest": [
+        {
+          "mode": "server",
+        ...
+        }
+      ] 
+    }
+    ```
+
+### Exchange Format Support
+In FHIR, resources are exchanged in the following formats: JSON, XML, and Turtle. Due to the popularity of JavaScript-based apps and ease of usage with JSON, the most popular exchange format for REST-styled APIs is JSON. 
+
+<p class="request-for-feedback">Input is requested on the appropriateness of mandating JSON or XML. Please comment by raising <a href="https://jira.hl7.org/projects/FHIR/issues">HL7 Jira Issues</a>.</p>
+
+### Mandatory Elements
+Mandatory elements are elements with minimum cardinality > 0. When an element is mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance see the guidance in AU Core on [Missing Data](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#missing-data). 
+
+An element can be both *Must Support* and mandatory, in which case the guidance for mandatory's Missing Data apply as described in AU Core [Missing Must Support and Mandatory Data](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#missing-must-support-and-mandatory-data).
+
+The convention in this guide is to mark all mandatory elements as *Must Support* unless they are nested under an optional element.
+
+### Must Support and Obligation
+Labelling an element *[Must Support](https://www.hl7.org/fhir/conformance-rules.html#mustSupport)* means that systems that produce or consume resources **SHALL** provide support for the element in some meaningful way. The FHIR standard does not define exactly what 'meaningful' support for an element means, but indicates that a profile **SHALL** make clear exactly what kind of support is required when an element is labelled as *Must Support*.
+
+Because AU eRequesting is a foundational standard, *Must Support* needs to be defined a way that does not impede or prescribe what a system does with the data, so as not to impede each implementation’s ability to tighten and define expectations for use under their own business rules, regulations, policies, etc. There is also a challenge that comes from inheritance of *Must Support* flags into implementation guides that have strict definitions for *Must Support* (e.g., must be able to display this value to an end user). AU eRequesting will only apply the *Must Support* flag on the elements that are necessary to support _minimum_ requirements and are expected to be flagged as *Must Support* across the majority of Australian FHIR implementation guides.
```

Comment:
missing word 'in':
..*Must Support* needs to be defined _<in>_ a way that...

Reply:
fixed